public/src/main/java/com/revenuecat/purchases/utils/JSONObjectParcelerDupe.kt

```diff
@@ -0,0 +1,17 @@
+package com.revenuecat.purchases.utils
+
+import android.os.Parcel
+import kotlinx.parcelize.Parceler
+import org.json.JSONObject
+
+/** @suppress */
+internal object JSONObjectParcelerDupe : Parceler<JSONObject> {
+
+    override fun create(parcel: Parcel): JSONObject {
+        return JSONObject(parcel.readString())
+    }
+
+    override fun JSONObject.write(parcel: Parcel, flags: Int) {
+        parcel.writeString(this.toString())
+    }
+}
```

Comment:
This is also temporary and will go away when removing the `public` module. Some classes in `common` and `public` depended on `JSONObjectParceler`, so I duplicated it for now